### PR TITLE
Upgrade to Akka 2.5.19

### PR DESCRIPTION
**Pull request by chbatey**
* Big version jump required as service discovery has moved into Akka
* Use akka-discovery
* Update docs

TODO
* Link to akka-discovery once akka 2.5.19 is released

### CODE REVIEW

File: docs/src/main/paradox/discovery/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+## Kubernetes API
```
Comment (raboof) [inline docs/src/main/paradox/discovery/kubernetes.md]: Great to split this up :+1:

**Comment by chbatey** [comment]
I'll get James's PRs in before merging this and conflicting it to the hizzle

**Comment by sebastianharko** [comment]
@raboof @chbatey can you guys wait a bit until you merge this, hoping that #405 gets merged before this, so I can avoid fixing a bunch of conflicts.

**Comment by patriknw** [comment]
Would be good to move forward with this and merge. I had some additional suggestions in #405. Would it be a lot of merge trouble if we merge this first?

**Comment by chbatey** [comment]
It'll cause a rename but should just be able to copy and paste the impl of the class. If it is much trouble I am happy to do it

**Pull request merged**